src/places/components/PlaceList.vue

```diff
@@ -97,6 +119,11 @@ export default {
 
   },
   computed: {
+    sortedPlaces () {
+      const subscribed = this.places.filter(e => e.isSubscribed)
+      const notSubscribed = this.places.filter(e => !e.isSubscribed)
+      return subscribed.concat(notSubscribed)
```

Comment:
Can use a sort.

Reply:
I had a try and felt that using `sort` is actually less readable, at least how I wrote it. Also `sort` is not necessary stable in JS, so we would take care not to lose the previous alphabetical sorting. Would leave as is.